tck/faces22/ajax/src/main/java/ee/jakarta/tck/faces/test/servlet30/ajax/filter/CustomResponseWriter.java

```diff
@@ -0,0 +1,585 @@
+/*
+ * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package ee.jakarta.tck.faces.test.servlet30.ajax.filter;
+
+import java.io.IOException;
+import java.io.Writer;
+
+import com.sun.faces.util.HtmlUtils;
+import com.sun.faces.util.MessageUtils;
```

Comment:
The TCK can't depend on com.sun.faces.util classes here.  Compilation fails for us.

Reply:
Right. Mistook them to be part of TCK utils.

Reply:
I've removed the unexpected Mojarra dependency altogether to prevent future mistakes like this and copied the HtmlUtils.